www/index.html

```diff
@@ -468,10 +468,13 @@ <h3>Display settings</h3>
                                     <select class="form-control" id="appThemeSelect">
```

Comment:
In my opinion,Ig we don't need to tell them this, as there won't be any auto options for OS/devices that can't handle dark mode.
Also I didn't clearly understand like to whom this note be displayed(to the available one or to the not available one or to everyone)

Reply:
I think we can do this succinctly. I presume @mossroy means that some people might think "Auto" means that the app will decide when to display dark theme and when to display light theme. But in reality, "Auto" means "whatever is set in the Operating System".

So all we want is a little note in the HTML that says "Auto themes match the dark/light mode of your device." I guess this note can be hidden if auto themes are not available.

This can be done with the same `querySelectorAll()` statement that you use, e.g. `querySelectorAll('option.auto, p.kiwix-auto-description')` (if you give that paragraph the class name `class="kiwix-auto-description"`).

Reply:
okay @Jaifroid  , I got it👍

Reply:
jaifroid, I'm implementing a code that'll display this note for 4seconds when the user selects auto theme and after 4secs this note will be removed from the DOM. Would that be okay?